inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -153,6 +158,11 @@ CNNNetworkNGraphImpl::CNNNetworkNGraphImpl(const ICNNNetwork& network) {
     }
 
     _ngraph_function = copyFunction(network.getFunction(), false);
+
+    ngraph::pass::Manager ssr_manager;
```

Comment:
I don't think that we should do it in this constructor. Because if ICNNNetwork contains the nGraph function it means that it has a type `CNNNetworkNGraphImpl` and it means that we already applied this transformation when we have created CNNNetwork from nGraph function.